README.md

```diff
@@ -1,39 +1,417 @@
-# holochain_serialization
+# Holochain Serialized Bytes
```

Comment:
Amazing level of documentation!

Reply:
@jost-s the docs were always there i just moved them out from somewhere hard to find 😅